meta-psa/README.md

```diff
@@ -0,0 +1,21 @@
+# meta-psa OpenEmbedded layer
+
+The meta-psa OpenEmbedded layer provides reference implementations and tests of
+the Arm Platform Security Architecture APIs.
+
+## PSA
+The Arm Platform Security Architecture (PSA) is a holistic set of threat
```

Comment:
Is that copied from the PSA site? It's a little marketingy.

Reply:
It does appear on the PSA site. It also appears at the start of each of the actual PSA spec documents and as an introductory statement at https://github.com/ARMmbed/mbed-crypto and https://github.com/ARM-software/psa-arch-tests. I'm open to different wording, but I don't feel qualified to summarize what PSA is myself.

Reply:
If that's how they phrased it - keep it